src/index.js

```diff
@@ -22,6 +22,7 @@ const defaults = {
   cacheKey,
   read,
   write,
+  compare,
```

Comment:
Maybe, naming things is hard right? I like the idea of refactoring this so that it is `validate` and that you have to pas the validation status into the callback, i.e. `true` if the cache is still valid. Thoughts?

Reply:
I think the `compare` name is accurate. It's what "sort" uses, for example, as a name: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort